shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IXHROverride.ts

```diff
@@ -0,0 +1,36 @@
+import { IPromise } from "@nevware21/ts-async";
+import { SendRequestReason } from "../applicationinsights-core-js";
+
+/** IPayloadData describes interface of payload sent via POST channel */
+export interface IPayloadData {
```

Comment:
Post Channel should now also be updated to reference this along with the enums above.

We may need to "re-export" these from the PostChannel extension to ensure that anyone attempt to reference from there picks up this common instance.